controllers/sandbox_controller.go

```diff
@@ -141,5 +213,6 @@ func (r *SandboxReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&sandboxv1alpha1.Sandbox{}).
 		Owns(&corev1.Pod{}, builder.WithPredicates(labelSelectorPredicate)).
+		Owns(&corev1.Service{}, builder.WithPredicates(labelSelectorPredicate)).
```

Comment:
I _thought_ there was a way to actually have the watch only watch objects with these labels, rather than just ignoring them .... do you know if there is?